medusa/subtitles.py

```diff
@@ -493,7 +492,7 @@ def save_subs(tv_episode, video, found_subtitles, video_path=None):
         if app.SUBTITLES_HISTORY:
             logger.debug(u'Logging to history downloaded subtitle from provider %s and language %s',
                          subtitle.provider_name, subtitle.language.opensubtitles)
-            history.logSubtitle(tv_episode, status, subtitle)
+            history.logSubtitle(tv_episode, subtitle)
```

Comment:
This is the only use case of `history.logSubtitle`, might as well fix the name to snake_case.